cmake/modules/VTA.cmake

```diff
@@ -79,6 +69,25 @@ elseif(PYTHON)
     target_link_libraries(vta ${__cma_lib})
   endif()
 
+  # TSIM-specific driver: we build when in either simulation modes
+  if(${VTA_TARGET} STREQUAL "sim" OR
+     ${VTA_TARGET} STREQUAL "tsim")
+    file(GLOB TSIM_RUNTIME_SRCS vta/src/*.cc)
+    list(APPEND TSIM_RUNTIME_SRCS vta/src/tsim/tsim_driver.cc)
+    list(APPEND RUNTIME_SRCS vta/src/dpi/module.cc)
+    add_library(tsim SHARED ${TSIM_RUNTIME_SRCS})
```

Comment:
what do you think on using `libvta_tsim` ? the thing is that using the simulation infrastructure is not general and everything is really tied to VTA. So, we can perhaps do something like `libvta_sim` and `libvta_tsim` I think using an explicit name is better.

Reply:
Thanks for the suggestion, I'm gonna go ahead with the new naming.